tests/ui/compiletest-self-test/aux-aux.rs

```diff
@@ -0,0 +1,8 @@
+//@ aux-crate: aux_aux_foo=aux_aux_foo.rs
+//@ aux-crate: aux_aux_bar=aux_aux_bar.rs
+//@ edition: 2021
+//@ compile-flags: --crate-type lib
+//@ check-pass
+
+use aux_aux_foo::Bar as IndirectBar;
+use aux_aux_bar::Bar as DirectBar;
```

Comment:
Please test that these types are the same

Reply:
Fixed!